test/integration/consul-container/libs/cluster/builder.go

```diff
@@ -20,9 +20,9 @@ const (
 	ConsulCACertKey = "consul-agent-ca-key.pem"
 )
 
-// BuildContext provides a reusable object meant to share common configuration settings
+// ClusterContext provides a reusable object meant to share common configuration settings
 // between agent configuration builders.
-type BuildContext struct {
+type ClusterContext struct {
```

Comment:
Trying out some renaming based on the discussion I had with @rboyer before the break. Suggestions welcome.